### PR TITLE
write docker setup at readme

### PR DESCRIPTION
**Pull request by avcuenes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,41 @@ Build the package
 ```
 colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to grid_map_geo
 ```
+## Docker Setup
+
+If your operating system doesn't support ROS 2 humble, docker is a great alternative.
+
+First of all, you have to build project and create image by copy and paste following command to terminal
```
Comment (Ryanf55) [inline README.md:33]:
```suggestion
First of all, you have to build the project and create an  image like so:
```

File: README.md [hunk 2]
```diff
@@ -26,6 +26,41 @@ Build the package
...
+
+```bash
+## Assuimg you are in the correct project directory
+docker build -t gmg .
+```
+To use shortcut , you may use following command;
```
Comment (Ryanf55) [inline README.md:39]:
```suggestion
To use a shortcut, you may use the following command:
```

File: README.md [hunk 3]
```diff
@@ -26,6 +26,41 @@ Build the package
...
+
+```bash
+## Assuimg you are in the correct project directory
+make docker_build
+```
+
+After create image, copy and paste the following command to terminal to run image;
```
Comment (Ryanf55) [inline README.md:46]:
```suggestion
After the image is created, copy and paste the following command to the terminal to run the image:
```

File: README.md [hunk 4]
```diff
@@ -26,6 +26,41 @@ Build the package
...
+
+```bash
+## Assuimg you are in the correct project directory
+docker run -it --net=host --ipc=host --privileged --env="DISPLAY"  --env="QT_X11_NO_MITSHM=1" --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" --volume="${XAUTHORITY}:/root/.Xauthority"  --entrypoint /bin/bash gmg
+```
+To use shortcut, you may use following commad;
```
Comment (Ryanf55) [inline README.md:52]:
```suggestion
To use a shortcut, you may use following command:
```

**Review by Ryanf55 — CHANGES_REQUESTED** [review]
Thanks for submitting the docs too! Just a few grammar/spelling things, otherwise LGTM.

**Comment by avcuenes** [comment]
> Thanks for submitting the docs too! Just a few grammar/spelling things, otherwise LGTM.

Thanks :)

**Pull request merged**